recipe/meta.yaml

```diff
@@ -41,8 +43,9 @@ test:
     - lighttpd -t -f lighttpd.conf
 
 about:
-  home: http://www.lighttpd.net/
-  license: BSD 3-Clause
+  home: https://www.lighttpd.net/
+  license: BSD-3-Clause
+  license_family: BSD
   license_file: COPYING
```

Comment:
Should we update the licence file? The correct name is Copyright

Reply:
COPYING is the correct file for v1.4 https://github.com/lighttpd/lighttpd1.4/blob/master/COPYING